plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -1076,10 +1077,11 @@ public void testQueryStringError()
     public void testAlias()
             throws IOException
     {
-        addAlias("orders", "orders_alias");
+        String aliasName = format("alias_%s", Math.abs(UUID.randomUUID().hashCode()));
```

Comment:
can we use `TestTable.randomTableSuffix()` here?

Reply:
Done